boards/arm/nrf52_blenano2/nrf52_blenano2.dts

```diff
@@ -52,10 +52,11 @@
 			reg = <0x00070000 0xD000>;
 		};
 
-		/*
-		 * The flash starting at 0x0007d000 and ending at
-		 * 0x0007ffff (sectors 125-127) is reserved for use
-		 * by the application.
-		 */
+#if defined(CONFIG_FS_FLASH_MAP_STORAGE)
```

Comment:
Fix this #if once https://github.com/zephyrproject-rtos/zephyr/pull/7341 will be resolved.

Reply:
done